packages/database/src/utilities/isMarkedChange.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+export const isMarkedChange = changeDetails => {
```

Comment:
Clever! And really helpful comment, maybe it should even go above the function

Reply:
I'd leave it there since its an implementation detail and not part of the function's contract - in the future, we could calculate this in a different way

Reply:
True! Perhaps a comment above that says 
```suggestion
// indicates whether a change notification was manually triggered, rather than a true database change
export const isMarkedChange = changeDetails => {
```
I feel the name may not be initially obvious so this sort of comment could be useful

Reply:
Good idea, will do!